### ISSUE TITLE
is there some difference  between paper with code?

### ISSUE THREAD

**Issue by fighterzhao** [ZZG-Z/Parallel-GAN#2]
In paper ,rec net have different lr in decoder and encoder,but code have same lr ,in paper a1=100,a2=10,but in code a1=10,a2=100

**Comment by ZZG-Z** [comment]
For question 1, the relevant code has been corrected(Although gains from varying learning rates are small ). 
For question 2, this is a typo in the paper, the correct values are a1=10 and a2=100 as shown in the code.
Thank you for your correction!!!

**Comment by fighterzhao** [comment]
Thank you for your answer. Your paper is worth my good study.

**Comment by fighterzhao** [comment]
in optical reconstruction task，the GAN loss shoule be calculated between fake optical image and real optical image,why do you concat SAR image with it ,for example ,fake_AB = torch.cat((self.real_A, self.fake_B), 1) is same in two task,in  translation task is reasonable，but in  reconstruction task,I think it is not reasonable.

**Comment by ZZG-Z** [comment]
Yes, the SAR image is useless, and remove it will not influence the reconstruction task, we add it just for align the recon and trans with cGan.



---- Replied Message ----
| From | ***@***.***> |
| Date | 12/14/2023 13:06 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [ZZG-Z/Parallel-GAN] is there some difference between paper with code? (Issue #2) |

in optical reconstruction task，the GAN loss shoule be calculated between fake optical image and real optical image,why do you concat SAR image with it ,for example ,fake_AB = torch.cat((self.real_A, self.fake_B), 1) is same in two task,in translation task is reasonable，but in reconstruction task,I think it is not reasonable.
